web-marketplace/src/lib/queries/react-query/bridge/getBridgeTxStatusQuery/getBridgeTxStatusQuery.ts

```diff
@@ -9,7 +9,9 @@ import {
 export const getBridgeTxStatusQuery = ({
   request,
   ...params
-}: ReactQueryBridgeTxStatusParams): ReactQueryBridgeTxStatusResponse => ({
+}: ReactQueryBridgeTxStatusParams & {
+  keepPreviousData?: boolean;
```

Comment:
why do you need this? it should be part of ReactQueryBridgeTxStatusParams already

in the latest version of react-query, keepPreviousData has been removed (in favor of placeholderData), so I suspect there might be some inconsistency in your node_modules, make sure to delete them all and reinstall